app/helpers/application_helper.rb

```diff
@@ -23,4 +23,26 @@ def render_flash_notice
       content_tag :p, flash[:notice], class: "notice"
     end
   end
+
+  def markdown(text)
+    options = {
+      filter_html:     true,
+      hard_wrap:       true,
+      link_attributes: { rel: 'nofollow', target: "_blank" },
+      space_after_headers: true,
+      fenced_code_blocks: true
+    }
+
+    extensions = {
+      autolink:           true,
+      superscript:        true,
+      disable_indented_code_blocks: true
+    }
+
+    @renderer ||= Redcarpet::Render::HTML.new(options)
```

Comment:
@lucasrenan it was the way I found to render markdown as html on view page. Feel free to suggest any improvement here. :)